index.html

```diff
@@ -41,60 +35,161 @@ <h2 class="primary_color">I'm a software developer</h2>
 
       <div>
         <p class="secondary_color">
-          I can help you build a product , feature or website Look
-          through some of my work and experience! <br />If you like
-          what you see and have a project you need coded, don't
+          I can help you build a product , feature or website Look through some of my work and
+          experience! <br />If you like what you see and have a project you need coded, don't
           hestiate to contact me.
         </p>
       </div>
 
       <ul class="social_icons_container">
         <li>
           <a href="https://twitter.com/marcealarconb">
-            <img
-              class="social-icon"
-              src="images/twitter.svg"
-              alt="Twitter contact"
-            />
+            <img class="social-icon" src="images/twitter.svg" alt="Twitter contact" />
           </a>
         </li>
         <li>
           <a href="https://www.linkedin.com/in/marceloalarconb/">
-            <img
-              class="social-icon"
-              src="images/linkedin.svg"
-              alt="Linkedin contact"
-            />
+            <img class="social-icon" src="images/linkedin.svg" alt="Linkedin contact" />
           </a>
         </li>
         <li>
           <a href="https://medium.com/@marceloalarconbarrenechea">
-            <img
-              class="social-icon"
-              src="images/medium.svg"
-              alt="Medium contact"
-            />
+            <img class="social-icon" src="images/medium.svg" alt="Medium contact" />
           </a>
         </li>
         <li>
           <a href="https://github.com/chelobotix/">
-            <img
-              class="social-icon"
-              src="images/github.svg"
-              alt="GitHub contact"
-            />
+            <img class="social-icon" src="images/github.svg" alt="GitHub contact" />
           </a>
         </li>
         <li>
           <a href="https://angel.co/u/marcelo-alarcon-1">
-            <img
-              class="social-icon"
-              src="images/angellist.svg"
-              alt="Angelist contact"
-            />
+            <img class="social-icon" src="images/angellist.svg" alt="Angelist contact" />
           </a>
         </li>
       </ul>
     </section>
+
+    <div class="d-flex justify-content-center">
+      <img src="images/arrow.png" alt="down arrow" />
+    </div>
+
+    <!-- Works Section -->
+
+    <section class="grid_5x1">
+      <div>
+        <h2 class="secondary_color">My Recent <br />Works</h2>
+      </div>
+
+      <article class="grid_5x1">
```

Comment:
- [ ] Kindly add a border to all the cards in the works section to match the figma template. You are expected to stick to the design template as much as possible.
### Your design
![1](https://user-images.githubusercontent.com/94551484/205329037-c0ba215b-0b21-4529-89b6-78e7f10c1724.png)
### Figma template
![f1](https://user-images.githubusercontent.com/94551484/205329042-31cdb5c5-25de-4688-a126-778e110960b1.png)

Reply:
:red_circle: As requested by the previous reviewer Please make sure you surround each card with a border to match the Figma design

Your Design
<img src="https://user-images.githubusercontent.com/94551484/205329037-c0ba215b-0b21-4529-89b6-78e7f10c1724.png">

Figma Design
<img src="https://user-images.githubusercontent.com/94551484/205329042-31cdb5c5-25de-4688-a126-778e110960b1.png" >